pkgs/applications/misc/onboard/default.nix

```diff
@@ -0,0 +1,134 @@
+{ config
+, pkgs
+, fetchurl
+, atspiSupport ? true, at_spi2_core ? null
+, gtk3
+, gnome3
+, libcanberra_gtk3
+, libudev
+, hunspell
+, isocodes
+, pkgconfig
+, xorg
+, libxkbcommon
+, python3
+, python35Packages
+, stdenv
+, bash
+, intltool
+, glib
+, gobjectIntrospection
+, gsettings_desktop_schemas
+, wrapGAppsHook
+}:
+
+python35Packages.buildPythonApplication rec {
+  name = "onboard-${version}";
+  majorVersion = "1.4";
+  version = "${majorVersion}.1";
+  src = fetchurl {
+    url = "https://launchpad.net/onboard/${majorVersion}/${version}/+download/${name}.tar.gz";
+    sha256 = "01cae1ac5b1ef1ab985bd2d2d79ded6fc99ee04b1535cc1bb191e43a231a3865";
+  };
+
+  doCheck = false;
```

Comment:
@FRidh So, I tried enabling tests and I'm getting the following error:

```
running install tests
found dconf version 0.26.0
Traceback (most recent call last):
  File "nix_run_setup.py", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 440, in <module>
    'build_ext': build_ext_custom},
  File "/nix/store/996yw9bc36jbwy0gn7jpaxvji8i0ijpx-python3.5-distutils-extra-2.39/lib/python3.5/site-packages/DistUtilsExtra/auto.py", line 88, in setup
    __dbus(attrs, src)
  File "/nix/store/996yw9bc36jbwy0gn7jpaxvji8i0ijpx-python3.5-distutils-extra-2.39/lib/python3.5/site-packages/DistUtilsExtra/auto.py", line 192, in __dbus
    lines = [l.strip() for l in open(f).readlines()]
  File "/nix/store/r7qpc32yr09l9a0d5y3b8i84kw5phx4p-python3-3.5.3/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 12: ordinal not in range(128)
builder for ‘/nix/store/grmh43c2vbw00cvpgrw4ahdgyk6jdxgn-onboard-1.4.1.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/abq3j75wbmgfpjvc5z0nb9y213hjdcwl-system-path.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/i1v54bpqcbigfqp5ab9npqmxvr25gf0y-nixos-system-atom-17.03.git.68b6c50b12M.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/i1v54bpqcbigfqp5ab9npqmxvr25gf0y-nixos-system-atom-17.03.git.68b6c50b12M.drv’ failed
```

It appears to be something [related to unicode](https://stackoverflow.com/questions/21129020/how-to-fix-unicodedecodeerror-ascii-codec-cant-decode-byte). Do you think this is related to the package or could it be related to NixOS's testing?